src/oc_erchef/apps/oc_chef_wm/src/chef_wm_named_sandbox.erl

```diff
@@ -159,7 +160,8 @@ commit_sandbox(Ctx, #chef_sandbox{org_id = OrgId, checksums = Checksums}=Sandbox
     %% prevent us from generating conflicts and error messages on the database.
     ChecksumsToCommit = [ C || {C, false} <- Checksums ],
     chef_db:mark_checksums_as_uploaded(Ctx, OrgId, ChecksumsToCommit),
-    chef_db:commit_sandbox(Ctx, Sandbox).
+    chef_db:commit_sandbox(Ctx, Sandbox),
```

Comment:
Can we move this logic -- including the call to `chef_db:commit_sandbox` into `chef_object_db:delete`? It is sort of the centralized place for deleting top-level objects including the database entry, any indexing, and authz entry.